manim/mobject/coordinate_systems.py

```diff
@@ -731,3 +739,303 @@ def get_coordinate_labels(self, *numbers, **kwargs):
+        x_max = config["frame_x_radius"] if (radius_max is None) else radius_max
+        x_line_frequency = self.azimuth_frequency * TAU
+
+        y_min = -config["frame_y_radius"] if (radius_max is None) else -radius_max
+        y_max = config["frame_y_radius"] if (radius_max is None) else radius_max
+        y_line_frequency = radius_line_frequency
+
+        super().__init__(
+            x_range=(x_min, x_max, x_line_frequency),
+            y_range=(y_min, y_max, y_line_frequency),
+            x_axis_config=radius_config,
+            color=color,
+            **kwargs,
+        )
+        self.azimuth_offset = azimuth_offset
+        self.azimuth_direction = azimuth_direction
+        self.azimuth_buff = azimuth_buff
+        self.azimuth_scale = azimuth_scale
+        self.prepare_for_nonlinear_transform()
+        self.background_lines.apply_function(
+            lambda p: np.array([p[0] * np.sin(p[1]), p[0] * np.cos(p[1]), 0])
+        )
+
+    def prepare_for_nonlinear_transform(self, num_inserted_curves=50):
+        for mob in self.background_lines.family_members_with_points():
+            num_curves = mob.get_num_curves()
+            if num_inserted_curves > num_curves:
+                mob.insert_n_curves(num_inserted_curves - num_curves)
+        return self
+
+    def polar_to_point(self, radius, azimuth):
+        r"""Gets a point from polar coordinates.
+
+        Parameters
+        ----------
+        radius : :class:`float`
+            The coordinate radius (:math:`r`).
+
+        azimuth : :class:`float`
+            The coordinate azimuth (:math:`\theta`).
+
+        Returns
+        -------
+        :class:`numpy.ndarray`
+            The point.
+        """
+        return self.coords_to_point(radius * np.cos(azimuth), radius * np.sin(azimuth))
+
+    def pr2pt(self, radius, azimuth):
+        """Abbreviation for ``polar_to_point``"""
+        return self.polar_to_point(radius, azimuth)
+
+    def point_to_polar(self, point):
+        r"""Gets polar coordinates from a point.
+
+        Parameters
+        ----------
+        point : :class:`numpy.ndarray`
+            The point.
+
+        Returns
+        -------
+        Tuple[:class:`float`, :class:`float`]
+            The coordinate radius (:math:`r`) and the coordinate azimuth (:math:`\theta`).
+        """
+        x, y = self.point_to_coords(point)
+        return np.sqrt(x ** 2 + y ** 2), np.arctan2(y, x)
+
+    def pt2pr(self, point):
+        """Abbreviation for ``point_to_polar``"""
+        return self.point_to_polar(point)
+
+    def get_coordinate_labels(self, r_vals, a_vals, **kwargs):
+        if r_vals is None:
+            r_vals = [
+                r for r in self.get_x_axis().default_numbers_to_display() if r >= 0
+            ]
+        if a_vals is None:
+            a_vals = np.arange(0, 1, self.azimuth_frequency)
+        r_mobs = self.get_x_axis().get_number_mobjects(*r_vals)
```

Comment:
```suggestion
    def get_coordinate_labels(self, r_vals, a_vals, **kwargs):
        if r_vals is None:
            r_vals = [r for r in self.get_x_axis().get_tick_range() if r >= 0]
        if a_vals is None:
            a_vals = np.arange(0, 1, self.azimuth_frequency)
        r_mobs = self.get_x_axis().add_numbers(r_vals)
```

Reply:
I think this is something `black` does, so `pre-commit.ci` might just revert it.

Reply:
There is also a difference in `get_tick_range()` and `.add_numbers()`  (which is why the docs are currently failing).

Reply:
Oh, sorry, didn't notice that. Was commenting on the expanded square brackets.

Reply:
![image](https://user-images.githubusercontent.com/25301457/116781094-99ea5200-aa78-11eb-8d7e-1d796db5b0e7.png)
 Still broken unfortunately.

Reply:
It should be fixed with #1430, where `NumberPlane` is restored to the previous usage. I've tested it locally with the changes and it works as expected.

Reply:
OK. I've been looking at adding `x_border`, `y_border` arguments to `NumberPlane` which can be used to fix some of the visual issues with `PolarPlane` namely not drawing the border gridline on one of the axes.